rotkehlchen/db/drivers/gevent.py

```diff
@@ -370,6 +371,7 @@ def write_ctx(self, commit_ts: bool = False) -> Generator['DBCursor', None, None
                         'INSERT OR REPLACE INTO settings(name, value) VALUES(?, ?)',
                         ('last_write_ts', str(ts_now())),
                     )
+                    settings.CachedSettings().update_entry('last_write_ts', ts_now())
```

Comment:
this is incorrect. You end up using different timestamp here and different in the line above. You should save it in a variable first and use it in both places.

Also ... instead of doing that, replace with a comment, and say that this particular settings is not cached knowingly.

This is a critical section code. Adding even one more function call can have very ugly and detrimental effects in the entire codebase as everything calls this .

Reply:
Should be fine. Can leave a note explaining why, and if we ever want to add it we can but I don't see it as a requirments. As this is read in only a single place, but we would be writingto the cache with extra logic in every single DB write.